src/auth/LoginForm.tsx

```diff
@@ -187,6 +199,10 @@ const LoginForm = ({
         <JSForm {...formProps} _internalId={_internalId} />
       </div>
     );
+  } else if (apexForm) {
```

Comment:
for my POC, I passed this prop in to LoginForm in hosted forms. for the full implementation we only want to pass it in for forms that are actually the apex form

do you think we identify that based on URL (lack of slug & feathery query params) or should a form have a flag indicating if it is the apex form?